From: Richard B Winters <rik@mmogp.com>
Date: Sun, 3 May 2015 15:11:35 -0400
Subject: Fix pcsd location and args in init script

 - pcsd on debian lives at /usr/share/pcsd, not /usr/lib/pcsd

Change-Id: I966db71f32256689a16310bbf0c6407a00abdf68
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/pcsd | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/pcsd/pcsd
+++ b/pcsd/pcsd
@@ -7,8 +7,8 @@
 
 ### BEGIN INIT INFO
 # Provides: pcsd
-# Required-Start: $network $syslog
-# Required-Stop: $network $syslog
+# Required-Start: $remote_fs $network $syslog
+# Required-Stop: $remote_fs $network $syslog
 # Should-Start: 
 # Should-Stop: 
 # Default-Start:
@@ -25,7 +25,7 @@
 
 exec="/usr/bin/ruby"
 prog="pcsd"
-config="/var/lib/pcsd"
+config="/usr/share/pcsd"
 
 [ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
 
@@ -34,9 +34,9 @@
 start() {
     [ -x $exec ] || exit 5
     echo -n $"Starting $prog: "
-    export GEM_HOME=/usr/lib/pcsd/vendor/bundle/ruby
+    #export GEM_HOME=/usr/lib/pcsd/vendor/bundle/ruby
     cd /var/lib/pcsd
-    daemon $exec -I/usr/lib/pcsd /usr/lib/pcsd/ssl.rb
+    daemon --command='/usr/bin/ruby -I/usr/share/pcsd /usr/share/pcsd/ssl.rb'
     # if not running, start it up here, usually something like "daemon $exec"
     retval=$?
     echo
